app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,73 @@
+
+$explore-background: #449662;
+$box-background: #abe3c0;
+$box-footer-background: #4d785d;
+.home {
+  .left-column {
+    @include span-columns(10);
+    margin-top: 20px;
+  }
+  .right-column {
+    @include span-columns(2);
+    margin-top: 20px;
+
+    h2 {
+      @include padding(20px);
+    }
+
+    li {
+      @include padding(20px);
+      border-bottom: 1px solid #EBECE7;
+    }
+  }
+
+  .lastest-books {
+    @include display(flex);
+    @include justify-content(space-around);
+    li {
+      z-index: 1;
+    }
+  }
+
+  .box {
+    @include border-left-radius(15px);
+    @include border-right-radius(15px);
+    position: relative;
+    background-color: $box-background;
+    .box-title {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3